app/controllers/users.js

```diff
@@ -132,21 +194,21 @@ exports.addDonation = function(req, res) {
       User.findOne({
         _id: req.user._id
       })
-      .exec(function(err, user) {
+        .exec(function(err, user) {
         // Confirm that this object hasn't already been entered
-        var duplicate = false;
-        for (var i = 0; i < user.donations.length; i++ ) {
-          if (user.donations[i].crowdrise_donation_id === req.body.crowdrise_donation_id) {
-            duplicate = true;
+          var duplicate = false;
+          for (var i = 0; i < user.donations.length; i++ ) {
+            if (user.donations[i].crowdrise_donation_id === req.body.crowdrise_donation_id) {
```

Comment:
I know this line may not exceed the airbnb 100-character limit, but it sure looks like it exceeds the Andela 80-character limit.